README.md

```diff
@@ -42,7 +42,7 @@ You can also find the Rdoc documentation on [Rubygems](https://rubygems.org/gems
 
 ## Requirements
 * Supported OS: Mac OS X, Linux, Windows
-* [Ruby](https://www.ruby-lang.org/en/downloads/) 2.2.2+
+* [Ruby](https://www.ruby-lang.org/en/downloads/) 2.3+
```

Comment:
It is not good Ruby upgrade is mixed to this PR. There are 2 ways how to fix it:
1 - stop support Ruby 2.2.x in separate PR, merge to master after review and then to actualize this branch
2 - remove changes related with migration to Ruby 2.3